bigquery/cloud-client/README.md

```diff
@@ -46,3 +46,45 @@ documentation](https://cloud.google.com/bigquery/create-simple-app-api):
 
     mvn exec:java -Dexec.mainClass=com.example.bigquery.SimpleApp
 
+## BigQuery + Logging SimpleApp
+
+This API Showcase demonstrates how to run a BigQuery query and then log some metrics to StackDriver Monitoring
+from the results, including the query runtime and number of rows returned.
+
+### Clone the sample app
+
+Copy the sample apps to your local machine, and cd to the bigquery/cloud-client directory:
+
+```
+git clone https://github.com/GoogleCloudPlatform/java-docs-samples
+cd java-docs-samples/bigquery/cloud-client
+```
+
+### Setup
+
+- Make sure [`gcloud`](https://cloud.google.com/sdk/docs/) is installed and initialized:
+```
+   gcloud init
+```
+- If this is the first time you are creating an App Engine project
+```
+   gcloud app create
+```
+- For local development, [set up][set-up] authentication
+- Enable [BigQuery][bigquery-api] and [Monitoring][monitoring-api] APIs
+- If you have not already enabled your project for StackDriver, do so by following [these instructions][stackdriver-setup].
+  
+[set-up]: https://cloud.google.com/docs/authentication/getting-started
+[bigquery-api]: https://console.cloud.google.com/launcher/details/google/bigquery-json.googleapis.com
+[monitoring-api]: https://console.cloud.google.com/launcher/details/google/monitoring.googleapis.com
+[stackdriver-setup]: https://cloud.google.com/monitoring/accounts/tiers#not-enabled
+
+### Run the SimpleApp
+
+Build your project with:
+
+	mvn clean package -DskipTests
+	
+Provide the projectId as a system variable when running the sample.
+
+    mvn exec:java -Dexec.mainClass=com.example.bigquery_logging.SimpleApp -DprojectId=<your-project-id>
```

Comment:
It looks like this is an appengine product - we generally suggest using maven to run/delpoy, ie `mvn appengine:run` and `mvn appengine:deploy`